packages/browser-wallet-api/package.json

```diff
@@ -7,7 +7,7 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@concordium/browser-wallet-api-helpers": "workspace:^",
-        "@concordium/common-sdk": "^6.3.0",
+        "@concordium/common-sdk": "^6.2.0",
```

Comment:
Is this right? If so, why?

Reply:
It is not. (I'm not sure how that happened)